tests/test_pinger.py

```diff
@@ -26,13 +28,17 @@ def test_read_status(self):
                 "70726F746F636F6C223A34347D7D"
             )
         )
-        status = self.pinger.read_status()
 
-        assert status.raw == {
-            "description": "A Minecraft Server",
-            "players": {"max": 20, "online": 0},
-            "version": {"name": "1.8-pre1", "protocol": 44},
-        }
+        with patch("mcstatus.status_response.JavaStatusResponse.build") as mock:
+            self.pinger.read_status()
+            mock.assert_called_once_with(
+                {
+                    "description": "A Minecraft Server",
+                    "players": {"max": 20, "online": 0},
+                    "version": {"name": "1.8-pre1", "protocol": 44},
+                }
+            )
+
```

Comment:
> This is no longer testing the output of the status function.

And it is shouldn't. All tests for `status` response objects are in `tests/status_response` folder, this is because response class moved out from the `pinger.py` file. We should keep tests only in one file for one goal, here - `pinger.py` contents.

> The `.raw` property is important and should remain present. Many people have written scripts based on its presence.

We already have [disscusion about the `raw` propery](https://github.com/py-mine/mcstatus/pull/306#discussion_r906827879), lets keep one disscusion for one theme.

> Instead of testing that `read_status()` calls this builder method, let's assert that the return value contains data the user would expect.

Testing output is much easier with mocking, instead of building this object without `build` method. You can look to tests in `tests/status_response` and what I do to test the `build` method.

Reply:
> And it is shouldn't

Ohh, I understood. It ~~should~~ must.

Reply:
Fixed in 0f7d43179b710bf2291536e98bd87724c3bd1714.